src/Aspire.Hosting/Publishing/DockerContainerRuntime.cs

```diff
@@ -116,16 +138,18 @@ private async Task<int> RunDockerBuildAsync(string contextPath, string dockerfil
         }
     }
 
-    public async Task BuildImageAsync(string contextPath, string dockerfilePath, string imageName, ContainerBuildOptions? options, CancellationToken cancellationToken)
+    public async Task BuildImageAsync(string contextPath, string dockerfilePath, string imageName, ContainerBuildOptions? options, Dictionary<string, string?> buildArguments, Dictionary<string, string?> buildSecrets, string? stage, CancellationToken cancellationToken)
     {
-        // Normalize the context path to handle trailing slashes and relative paths
```

Comment:
This comment still seems valuable. Is there a reason to delete it?

Reply:
Oopsie during rebase :'(